### PR TITLE
8334305: Remove all code for  nsk.share.Log verbose mode

### PR DESCRIPTION
**Pull request by rsunderbabu**
Cleaning up nsk.share.Log code after the verbose mode was set always true.

Tested all the vmTestbase/ tests.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8334305](https://bugs.openjdk.org/browse/JDK-8334305): Remove all code for  nsk.share.Log verbose mode (**Enhancement** - P4)

### Reviewers
 * [Leonid Mesnik](https://openjdk.org/census#lmesnik) (@lmesnik - **Reviewer**) Review applies to [a346de1c](https://git.openjdk.org/jdk/pull/21267/files/a346de1ce3387f99bb3f8d63c64b9c6bc1212bcb)
 * [Hamlin Li](https://openjdk.org/census#mli) (@Hamlin-Li - **Reviewer**)
 * [Chris Plummer](https://openjdk.org/census#cjplummer) (@plummercj - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/21267/head:pull/21267` \
`$ git checkout pull/21267`

Update a local copy of the PR: \
`$ git checkout pull/21267` \
`$ git pull https://git.openjdk.org/jdk.git pull/21267/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 21267`

View PR using the GUI difftool: \
`$ git pr show -t 21267`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/21267.diff">https://git.openjdk.org/jdk/pull/21267.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/21267#issuecomment-2383460447)

### CODE REVIEW

File: test/hotspot/jtreg/vmTestbase/nsk/share/Log.java
Comment (plummercj) [inline test/hotspot/jtreg/vmTestbase/nsk/share/Log.java:264]: Need replace the comma with a period.

File: test/hotspot/jtreg/vmTestbase/nsk/share/Log.java
Comment (plummercj) [inline test/hotspot/jtreg/vmTestbase/nsk/share/Log.java:287]: Is this method ever called? Is there a CR to remove it (and any references to it)?

File: test/hotspot/jtreg/vmTestbase/nsk/share/Log.java
Comment (plummercj) [inline test/hotspot/jtreg/vmTestbase/nsk/share/Log.java:340]: Need period at end of sentence.

File: test/hotspot/jtreg/vmTestbase/nsk/share/Log.java
Comment (plummercj) [inline test/hotspot/jtreg/vmTestbase/nsk/share/Log.java:342]: Does it really do all this? It looks to me like it just switches to the new stream. I'm not sure what is meant by "error summary" and cancelling.

File: test/hotspot/jtreg/vmTestbase/nsk/share/Log.java
Comment (rsunderbabu) [reply test/hotspot/jtreg/vmTestbase/nsk/share/Log.java:287]: These are deprecated methods and are still getting called. The calls need to be replaced with display. I will create a separate CR for this and address because the impact radius of such a change is bigger than this CR.

**Comment by dholmes-ora** [comment]
@rsunderbabu  this should wait for the second review by @plummercj  to be completed before it is integrated. Thanks.

**Comment by rsunderbabu** [comment]
> @rsunderbabu this should wait for the second review by @plummercj to be completed before it is integrated. Thanks.

Thanks for pointing out David. I missed seeing those comments. I have addressed them now.

File: test/hotspot/jtreg/vmTestbase/nsk/share/Log.java
Comment (plummercj) [inline test/hotspot/jtreg/vmTestbase/nsk/share/Log.java:40]: Missing period.

File: test/hotspot/jtreg/vmTestbase/nsk/share/Log.java
Comment (plummercj) [reply test/hotspot/jtreg/vmTestbase/nsk/share/Log.java:287]: ok

File: test/hotspot/jtreg/vmTestbase/nsk/share/Log.java
Comment (plummercj) [reply test/hotspot/jtreg/vmTestbase/nsk/share/Log.java:342]: Ok, but unless it is going to be removed by another CR soon, I think the comments should at least reflect what it currently does.

**Comment by rsunderbabu** [comment]
JDK-8341302 created for addressing deprecated method usage.

**Review by Hamlin-Li — APPROVED** [review]

**Pull request closed without merging**
